### PR TITLE
Asser-M365DSCBlueprint: add additional Auth Meth to Asser-M365DSCBlueprint

### PR DESCRIPTION
**Pull request by Timsto**

### CODE REVIEW

**Review by ykuijs — CHANGES_REQUESTED** [review]
Could you please also update the changelog to reflect this change?

**Comment by ykuijs** [comment]
@NikCharlebois Can you please also check this PR? Not 100% if this will work for this function.

**Comment by NikCharlebois** [comment]
@Timsto This function should not handle credentials. It is purely meant to retrieve the domain if service principal if passed. If credentials are used, then determining the domain is very easy and straightforward:

$domain = $credentials.UserName.Split('@')[1]

**Comment by andikrueger** [comment]
@NikCharlebois  and @Timsto Are there any updates to this PR? 

At the moment there are two tasks open:

- [ ] Update Changelog
- [ ] Remove `Credential` parameter from `Get-M365DSCTenantDomain`

File: Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1 [hunk 1]
```diff
@@ -1153,6 +1153,10 @@ Internal
 function Get-M365DSCTenantDomain
 {
     param(
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
```
Comment (andikrueger) [inline Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1:1159]: as mentioned in https://github.com/microsoft/Microsoft365DSC/pull/1792#issuecomment-1061727808 this function should not handle any Credentials.

**Comment by NikCharlebois** [comment]
@Timsto have you had a chance to look at the comments? Should we keep this PR open or close it?

**Comment by ykuijs** [comment]
Closing this PR due to inactivity, but code will be adopted in a different PR I am working on.

**Pull request closed without merging**